nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -109,6 +109,15 @@
       without Syncthing resetting the permission on every start.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>ntp</literal> module now has sane default restrictions.
```

Comment:
Can you also mention the DNS reflector stuff here?

Reply:
It's not a very informative message. It should either be removed or it should be rewritten with the intention to actually help someone. I.e., someone would be wondering "why was it not sane before?".

If there is *no* case in which this will break for someone, then remove the message. If there is a case in which it can break, describe it as an incompatible change with specifics.